app/components/SocialButton.tsx

```diff
@@ -40,6 +41,18 @@ export function SocialButton(props: SocialButtonProps) {
   )
 }
 
+interface SocialButtonsProps {
+  socialButtons: Array<SocialButtonProps>
+}
+
+export const SocialButtons = ({ socialButtons }: SocialButtonsProps) => (
+  <>
+    {socialButtons.map((socialButtonProps) => (
```

Comment:
In the cases where we don't have a URL, should we filter those out?

Reply:
that happens above on line 29. didn't want to replicate it

Reply:
my bad, the `key` should be `${icon}-${index}`. updated here: https://github.com/infinitered/ChainReactApp2023/pull/151/commits/fbcadd2d14fd06e79d18448f6a00ac5a8b066acd